app/src/main/java/com/kickstarter/ui/adapters/DiscoveryPagerAdapter.java

```diff
@@ -40,8 +41,12 @@ public void setPrimaryItem(final @NonNull ViewGroup container, final int positio
   }
 
   @Override
-  public @NonNull Fragment getItem(final int position) {
-    return DiscoveryFragment.newInstance(position);
+  public
+  @NonNull
+  Fragment getItem(final int position) {
```

Comment:
we keep all of the modifiers in the same line of the function.

did something in our check style say to do this?